components/shindig-wso2-features/src/main/javascript/wso2features/identity/identity.js

```diff
@@ -34,6 +34,27 @@ wso2.gadgets.identity = (function () {
     var RPC_SERVICE_GET_ACCESS_TOKEN = 'RPC_SERVICE_GET_ACCESS_TOKEN';
 
     /**
+     * Service name to get logged-in user domain.
+     * @const
+     * @private
+     */
+    var RPC_SERVICE_GET_TENANT_DOMAIN = 'RPC_SERVICE_GET_TENANT_DOMAIN';
+
+    /**
+     * Service name to get hostname.
+     * @const
+     * @private
+     */
+    var RPC_SERVICE_SERVER_HOSTNAME = 'RPC_SERVICE_SERVER_HOSTNAME';
```

Comment:
Hostname, and port are not something related to identity, therefore we need to move this to some other gadget feature. Lets create a new feature called 'server' and include these methods there.